fuzzers/lafintel/builder.Dockerfile

```diff
@@ -28,6 +28,9 @@ RUN wget https://lcamtuf.coredump.cx/afl/releases/afl-2.26b.tgz -O /afl-2.26b.tg
     tar xvzf /afl-2.26b.tgz -C / && \
     mv /afl-2.26b /afl && \
     cd /afl && \
+    git clone https://github.com/google/AFL.git /afl/recent_afl && \
```

Comment:
We need to use a specific revision, even for latest afl, e.g. see
https://github.com/google/fuzzbench/blob/master/fuzzers/afl/builder.Dockerfile#L22
That makes it easy to know what was benchmarked against.